src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    public static final int amountOfFigures = 5;
```

Comment:
All magic numbers in your code should be constants. However, the `amountOfFigures` constant seems to be misplaced here, as it does not pertain to the properties of a `Figure`. It would be better suited in the `FigureSupplier` class where it is used for logic related to figure generation.